include/picongpu/plugins/particleCalorimeter/ParticleCalorimeter.hpp

```diff
@@ -527,9 +528,8 @@ namespace picongpu
             , m_cellDescription(cellDescription)
             , leftParticlesDatasetName("calorimeterLeftParticles")
         {
-            foldername = m_help->getOptionPrefix() + "/" + m_help->filter.get(m_id);
-            filenamePrefix
-                = m_help->getOptionPrefix() + "_" + m_help->fileName.get(m_id) + "_" + m_help->filter.get(m_id);
+            foldername = m_help->getOptionPrefix() + "/";
+            filenamePrefix = m_help->getOptionPrefix() + "_" + m_help->filter.get(m_id);
```

Comment:
I was under the impression that we have kept the name so far for compatibility with the naming that the Python post-processing expects. Is that correct? If yes, do we need to update the naming there?

Reply:
So far, there was no python post-processing. I am currently in the process of adding it #4697